vars/rpmDashboardsDistValidation.groovy

```diff
@@ -58,7 +58,7 @@ def call(Map args = [:]) {
 
     //Start the installed OpenSearch-Dashboards distribution
     systemdCommands(
-            command: "start",
+            command: "sudo env OPENSEARCH_INITIAL_ADMIN_PASSWORD=myStrongPassword123! start",
```

Comment:
This will result in `sudo systemctl sudo env OPENSEARCH_INITIAL_ADMIN_PASSWORD=myStrongPassword123! start`

Reply:
don't think we need to pass variable here